kcidb/issue_editor/templates/index.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
```

Comment:
Hmm, this file isn't really a template, but is just a pure HTML file, so I don't think it should be under that directory. We can simplify things if we just put it into `kcidb/issue_editor`.